### ISSUE TITLE
Fix MacOS code-signing for DMG packages

### ISSUE THREAD

**Issue by greendog99** [thomasnordquist/MQTT-Explorer#76]
Downloaded MQTT-Explorer-0.2.3.dmg, opened DMG archive, launched app, immediately got "MQTT Explorer quit unexpectedly" pop-up. Tried on another Mac (which has never had MQTT-Explorer installed before), same problem. Report included below.

Thanks for all you do! I love MQTT-Explorer...

```
Process:               MQTT Explorer [3885]
Path:                  /Users/USER/Downloads/MQTT Explorer.app/Contents/MacOS/MQTT Explorer
Identifier:            de.t7n.apps.mqtt-explorer
Version:               ???
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           MQTT Explorer [3885]
User ID:               501

Date/Time:             2019-03-07 09:52:19.365 -0800
OS Version:            Mac OS X 10.14.3 (18D109)
Report Version:        12
Bridge OS Version:     3.0 (14Y674)
Anonymous UUID:        77A10C44-DA6E-1B13-FFD5-AE8FCF785A2E

Time Awake Since Boot: 60000 seconds

System Integrity Protection: enabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (Code Signature Invalid)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    Namespace CODESIGNING, Code 0x1

kernel messages:

VM Regions Near 0 (cr2):
--> 
    __TEXT                 0000000107c37000-0000000107c38000 [    4K] r-x/rwx SM=COW  

Thread 0 Crashed:
0                                 	0x0000000114c0c000 _dyld_start + 0

Thread 0 crashed with X86 Thread State (64-bit):
  rax: 0x0000000000000000  rbx: 0x0000000000000000  rcx: 0x0000000000000000  rdx: 0x0000000000000000
  rdi: 0x0000000000000000  rsi: 0x0000000000000000  rbp: 0x0000000000000000  rsp: 0x00007ffee7fc8b90
   r8: 0x0000000000000000   r9: 0x0000000000000000  r10: 0x0000000000000000  r11: 0x0000000000000000
  r12: 0x0000000000000000  r13: 0x0000000000000000  r14: 0x0000000000000000  r15: 0x0000000000000000
  rip: 0x0000000114c0c000  rfl: 0x0000000000000200  cr2: 0x0000000000000000
  
Logical CPU:     0
Error Code:      0x00000000
Trap Number:     0


Binary Images:
       0x107c37000 -        0x107c37ffb + (0) <6E272AEB-51ED-363B-B2D0-1E8B0C094EDF> 
       0x114c0b000 -        0x114c89a87 + (655.1) <3EBA447F-A546-366B-B302-8DC3B21A3E30> 

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 37055
    thread_create: 0
    thread_set_state: 0

VM Region Summary:
ReadOnly portion of Libraries: Total=688K resident=0K(0%) swapped_out_or_unallocated=688K(100%)
Writable regions: Total=8400K written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=8400K(100%)
 
                                VIRTUAL   REGION 
REGION TYPE                        SIZE    COUNT (non-coalesced) 
===========                     =======  ======= 
STACK GUARD                       56.0M        2 
Stack                             8192K        2 
__DATA                             232K        4 
__LINKEDIT                         176K        3 
__TEXT                             512K        3 
shared memory                        8K        3 
===========                     =======  ======= 
TOTAL                             64.9M       11 

Model: MacBookPro13,3, BootROM 251.0.0.0.0, 4 processors, Intel Core i7, 2.7 GHz, 16 GB, SMC 2.38f7
Graphics: Intel HD Graphics 530, Intel HD Graphics 530, Built-In
Graphics: Radeon Pro 455, AMD Radeon Pro 455, PCIe
Memory Module: BANK 0/DIMM0, 8 GB, LPDDR3, 2133 MHz, 0x802C, 0x4D5435324C31473332443450472D30393320
Memory Module: BANK 1/DIMM0, 8 GB, LPDDR3, 2133 MHz, 0x802C, 0x4D5435324C31473332443450472D30393320
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x15A), Broadcom BCM43xx 1.0 (7.77.61.1 AirPortDriverBrcmNIC-1305.2)
Bluetooth: Version 6.0.10f1, 3 services, 27 devices, 1 incoming serial ports
Network Service: Wi-Fi, AirPort, en0
PCI Card: pci1b73,1100, USB eXtensible Host Controller, Thunderbolt@196,0,0
USB Device: USB 3.0 Bus
USB Device: iBridge
USB Device: USB 3.0 Bus
USB Device: USB3.1 Hub
USB Device: Hub
USB Device: LG UltraFine Display Camera
USB Device: USB2.1 Hub
USB Device: Hub
USB Device: USB Controls
USB Device: USB Audio
Thunderbolt Bus: MacBook Pro, Apple Inc., 39.3
Thunderbolt Bus: MacBook Pro, Apple Inc., 39.3
Thunderbolt Device: UltraFine 5K, LG Electronics, 3, 9.4
```

**Comment by thomasnordquist** [comment]
Thanks for reporting, something went wrong with the code signing.
The AppStore build however works.
Don't be alarmed, the default mqtt-brokers seem to have some trouble at the moment.

I removed the .dmg build until fixed.

**Comment by greendog99** [comment]
Thanks! I wasn't sure if the App Store was the latest version (there's no version number or release date listed there). Downloaded it from there.

**Comment by thomasnordquist** [comment]
I also re-uploaded a unsigned version.
I'll try to keep the store releases always up-to-date / release them earlyer so all releases are in sync.

**Comment by thomasnordquist** [comment]
Fixed with 9e71c3132eda90f855746059413eae8c67cc8366
